omocha-domain/src/main/java/org/omocha/domain/review/ReviewServiceImpl.java

```diff
@@ -67,13 +67,13 @@ private Auction validateAuctionStatus(Long auctionId) {
 
 	private Long determineRecipientId(Long reviewerId, Auction auction) {
 		Conclude conclude = concludeReader.getConclude(auction.getAuctionId());
-		Long sellerId = auction.getMemberId();
-		Long buyerId = conclude.getBuyer().getMemberId();
+		Long sellerMemberId = auction.getMemberId();
+		Long buyerMemberId = conclude.getBuyer().getMemberId();
 
-		if (reviewerId.equals(sellerId)) {
-			return buyerId;
-		} else if (reviewerId.equals(buyerId)) {
-			return sellerId;
+		if (reviewerId.equals(sellerMemberId)) {
+			return buyerMemberId;
+		} else if (reviewerId.equals(buyerMemberId)) {
+			return sellerMemberId;
 		} else {
 			throw new ReviewPermissionDeniedException(auction.getAuctionId(), reviewerId);
 		}
```

Comment:
이 if-else 구문에서 else if 나 else 는 사실 의미가 없어보이는데요 아래처럼 바꿔도 동일하게 동작하겠네요
어차피 같은 로직이라.. 바꾸라는건 아니고 해덕님이 보시고 더 가독성이 좋다고 판단되는걸로 가주세요

```suggestion
if(reviewerId.equals(sellerMemberId)) {
  return buyerMemberId;
}

if(reviewerId.equals(buyerMemberId)) {
  return sellerMemberId;
}

throw new ReviewPermissionDeniedException(auction.getAuctionId(), reviewerId);
```

Reply:
아 이건 exception throw 때문에 else를 사용한건데
생각해보니 위에 return을 안거치면 마지막에 그냥 exception을 throw 하면 되는거였네요;;

저도 저거 else 쓰는걸 선호하지 않는데 답변 감사합니다!